packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/hierarchy/HierarchyLifecycleControllerTests.java

```diff
@@ -14,6 +14,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertFalse;
```

Comment:
- assertThat seems to become an unused import
- It's more powerful to rely on assertThat instead of junit assertions so I'll just keep using assertj